content/variables/git.md

```diff
@@ -19,6 +19,8 @@ toc: false
 wip: false
 ---
 
+This variable shows metada about a specific piece of content. It is useful for times when you are using Git and Hugo to manage a project or in a multi-author setting, for example. It will not be useful as a cache busting method.
```

Comment:
It's come up on other issues and the forum (i.e. cache busting), but I agree we don't need to add "Don't use X for this" for every Hugo feature. My suggestion was more about the language and placement :smile: